discovery-provider/alembic/versions/7b3f27e78b84_add_credits_table.py

```diff
@@ -0,0 +1,33 @@
+"""add-credits-table
+
+Revision ID: 7b3f27e78b84
+Revises: c64edfb319a3
+Create Date: 2020-04-29 12:53:06.806762
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '7b3f27e78b84'
+down_revision = 'c64edfb319a3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('credits',
+        # A child track "credits" parent track (and can credit many)
+        sa.Column('parent_track_id', sa.Integer(), nullable=False, index=True),
+        sa.Column('child_track_id', sa.Integer(), nullable=False, index=True),
+        # TODO: Consider a way to make this possible. It's not right now because
+        # there is no unique constraint on tracks.track_id
+        # sa.ForeignKeyConstraint(['parent_track_id'], ['tracks.track_id'], ),
```

Comment:
i don't think this wouldn't matter since tracks is append only and we never delete. Even if we could have foreign key constraints, a one way foreign key constraint with tracks would be okay.  But, we definitely would not want a two way foreign key constraint between this and tracks.

Reply:
yes -- but the benefit is that you wouldn't be allowed to insert garbage into this table, which we can't enforce at all now.